### PR TITLE
Add pipes for strict template check

### PR DESCRIPTION
**Pull request by iozbeyli**
Add artemis-time-ago and artemis-translate pipes for strict template checks.

### CODE REVIEW

**Review by ivanchimeno — DISMISSED** [review]

**Review by FuchsDominik — DISMISSED** [review]
Overflew it and could not find any problems

**Review by ivanchimeno — DISMISSED** [review]

File: docs/dev/guidelines/client.rst [hunk 1]
```diff
@@ -353,6 +353,19 @@ When creating a completely new route you will have to register the new paths in
...
 
+12. Strict Template Check
+===================================
+
+To prevent errors for strict template rule in TypeScript, Artemis uses following approaches.
+
+Use ArtemisTranslatePipe instead of TranslatePipe
+**************************************************
+- placeholder="{{ 'global.form.newpassword.placeholder' | translate }}"
++ placeholder="{{ 'global.form.newpassword.placeholder' | artemisTranslate }}"
```
Comment (FrankeLukas) [inline docs/dev/guidelines/client.rst:364]: GitHub Actions complains here: `Bullet list ends without a blank line; unexpected unindent.`

File: docs/dev/guidelines/client.rst [hunk 2]
```diff
@@ -353,6 +353,19 @@ When creating a completely new route you will have to register the new paths in
...
 
+Use ArtemisTimeAgoPipe instead of TimeAgoPipe
+*********************************************
+- <span [ngbTooltip]="submittedDate | artemisDate">{{ submittedDate | amTimeAgo }}</span>
++ <span [ngbTooltip]="submittedDate | artemisDate">{{ submittedDate | artemisTimeAgo }}</span>
```
Comment (FrankeLukas) [inline docs/dev/guidelines/client.rst:369]: Same here.

**Review by FrankeLukas — APPROVED** [review]
Still deployed and started without errors on TS1.

File: docs/dev/guidelines/client.rst (hunk 1 above)
Comment (iozbeyli) [reply docs/dev/guidelines/client.rst]: Fixed it

File: docs/dev/guidelines/client.rst (hunk 2 above)
Comment (iozbeyli) [reply docs/dev/guidelines/client.rst]: Fixed it

**Review by ivanchimeno — APPROVED** [review]

**Review by nicolasruscher — APPROVED** [review]
As the changes are huge but mostly the same, I looked over them, could not find any issues. code lgtm 👍

**Review by krusche — APPROVED** [review]

**Pull request merged**